opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -170,11 +170,21 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
     { name: 'user_id', type: 'string', mandatoryType: 'no', multiple: false, upsert: false },
     { name: 'token', type: 'string', mandatoryType: 'external', multiple: false, upsert: false },
     { name: 'stream_id', type: 'string', mandatoryType: 'external', multiple: false, upsert: false },
-    { name: 'running', type: 'string', mandatoryType: 'no', multiple: false, upsert: false },
+    { name: 'running', type: 'boolean', mandatoryType: 'no', multiple: false, upsert: false },
     { name: 'current_state_date', type: 'date', mandatoryType: 'no', multiple: false, upsert: false },
     { name: 'listen_deletion', type: 'boolean', mandatoryType: 'external', multiple: false, upsert: false },
     { name: 'no_dependencies', type: 'boolean', mandatoryType: 'external', multiple: false, upsert: false },
   ],
+  [ENTITY_TYPE_HISTORY]: [
+    { name: 'event_type', type: 'string', mandatoryType: 'internal', multiple: false, upsert: false },
+    { name: 'event_scope', type: 'string', mandatoryType: 'internal', multiple: false, upsert: false },
+    { name: 'user_id', type: 'string', mandatoryType: 'internal', multiple: false, upsert: false },
+    { name: 'applicant_id', type: 'string', mandatoryType: 'no', multiple: false, upsert: false },
+    { name: 'group_ids', type: 'string', mandatoryType: 'internal', multiple: true, upsert: false },
+    { name: 'organization_ids', type: 'string', mandatoryType: 'internal', multiple: true, upsert: false },
+    { name: 'timestamp', type: 'date', mandatoryType: 'internal', multiple: false, upsert: false },
+    { name: 'context_data', type: 'dictionary', mandatoryType: 'internal', multiple: false, upsert: false },
+  ]
```

Comment:
I think there are also `ENTITY_TYPE_WORK`, `ENTITY_TYPE_ACTIVITY`, `ENTITY_TYPE_FEED` that are missing attributes definition